Dockerfile

```diff
@@ -1,17 +1,11 @@
-FROM golang:1.13.5-buster
+FROM golang:1.14 AS builder
+WORKDIR /var/tmp/wapi
+COPY . /var/tmp/wapi
+RUN CGO_ENABLED=0 GOOS=linux go build -o /var/tmp/wapi/bin/wapi /var/tmp/wapi/main.go
 
+FROM alpine:3.12
 MAINTAINER Roma Erema
-
-COPY . /wapi_build
-
-ENV GOPATH /wapi_build
-
-RUN echo "building wapi..."
-RUN cd /wapi_build \
-    && go build -o /wapi/build main.go \
-    && rm -rf /wapi_build \
-RUN echo "ok"
-
-WORKDIR /wapi
-
-CMD ./build
+WORKDIR /root/
+COPY --from=builder /var/tmp/wapi/bin/wapi .
+COPY ./.env .
+CMD ["./wapi"]
```

Comment:
Looks good, but you need to start your application with env variables initialized. I don't see any initialization